index.html

```diff
@@ -0,0 +1,198 @@
+            <br>
+            <div class="buttons">
+                <ul class="list">
+                    <li><div class="tag">html</div></li>
+                    <li><div class="tag">css</div></li>
+                    <li><div class="tag">javascript</div></li>
+                </ul>
+                <button>See Project</button>
+            </div>
+        </section>
+    </section>
+    <section class="white-bg btl-rad">
+        <section class="about">
+            <br>
+            <h1 class="main-text">About Myself</h1>
+            <p class="content">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me..</p>
+            <br>
+            <p>LET'S CONNECT</p>
+            <br>
+            <div class="contact">
+                <ul class="list">
+                    <li><a href="#"><img src="images/vector-1.png" alt="vector1"></a></li>
+                    <li><a href="#"><img src="images/vector-2.png" alt="vector2"></a></li>
+                    <li><a href="#"><img src="images/vector-3.png" alt="vector3"></a></li>
+                    <li><a href="#"><img src="images/vector-4.png" alt="vector4"></a></li>
+                    <li><a href="#"><img src="images/vector-5.png" alt="vector5"></a></li>
+                </ul>
+                <button>Get My Resume</button>
+            </div>
+        </section>
+    </section>
+    <section class="white-bg">
+        <section class="tools">
+            <div class="languages">
+                <ul class="coder">
+                    <li><a href="#"><img src="images/languages.png" alt="languages"></a></li>
+                    <li>
+                        <ul class="lang-container">
+                            <li class="lang">
+                                <a href="#"><img class="lang-logo" src="images/javasript-logo.png" alt="javascript"></a>
+                                <p class="upper">Javascript</p>
+                            </li>
+                            <li class="lang">
+                                <a href="#"><img class="lang-logo" src="images/html-logo.png" alt="html"></a>
+                                <p class="upper">Html</p>
+                            </li>
+                            <li class="lang">
+                                <a href="#"><img class="lang-logo" src="images/css-logo.png" alt="css"></a>
+                                <p class="upper">css</p>
+                            </li>
+                        </ul>
+                    </li>
+                    <li><a href="#"><img src="images/frameworks.png" alt="frameworks"></a></li>
+                    <li><a href="#"><img src="images/skills.png" alt="skills"></a></li>
+                </ul>
+            </div>
+        </section>
+    </section> 
```

Comment:
- [ ] [OPTIONAL] You should add the margin/padding, flex, or other techniques in this sections as well to have a more professional portfolio and applying responsive design 😉 

Figma design | Yours

![image](https://user-images.githubusercontent.com/60164239/129457660-333284ec-fe1f-4906-bdfc-c143ee6f1aa5.png)